aptos-move/aptos-vm/src/access_path_cache.rs

```diff
@@ -1,99 +0,0 @@
-// Copyright © Aptos Foundation
```

Comment:
I remember vaguely that this thing was added (during the Diem days) due to some very noticeable performance drops. Can you figure out if this is still needed from a perf standpoint?

Reply:
We probably should use it, but no one touched that for 2 years, so I think we should revisit this separately. But otherwise yes, I can benchmark to see if we spent time in access path construction.

Reply:
We definitely need a solution for gigantic state keys and access paths floating around, but this may not be it.